### DISCUSSION TITLE
Troubleshoot to node-schedule

### DISCUSSION THREAD

**Discussion by riddhi199** [community/community#29779] [Actions] [answered]
We have using "node-schedule" package for our running application to schedule notification to our users  but somehow notification cron job is not working properly, sometimes it stops and we got error message like below:

roundUpCron/scheduleNotification Error: CronDate: unhandled timestamp: 1659539835855
    at new CronDate (/app/node_modules/cron-parser/lib/date.js:233:11)
    at RecurrenceRule._nextInvocationDate (/app/node_modules/node-schedule/lib/Invocation.js:137:15)
    at scheduleNextRecurrence (/app/node_modules/node-schedule/lib/Invocation.js:324:56)
    at Job.schedule (/app/node_modules/node-schedule/lib/Job.js:266:13)
    at Object.scheduleJob (/app/node_modules/node-schedule/lib/schedule.js:39:11)
    at Object.roundUpCron.scheduleNotification (/app/modules/v1/cron/roundUpCron.js:76:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

Could you please revert?

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/node-schedule/node-schedule? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:

**Comment by queenofcorgis** [comment]
Expanding on @airtower-luna's comment, if you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
